website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -194,6 +196,20 @@ const useExternalApiFetcher = () => {
 };
 ```
 
+Available options are:
```

Comment:
Perhaps we could add a link to the new section that explains how to use this header (`/custom-applications/concepts/integrate-with-your-own-api#configuring-the-audience-policy`).